modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -6,9 +6,18 @@
 [id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
 = Configuring the PROXY protocol for an Ingress Controller
 
-A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[the PROXY protocol] when an Ingress Controller uses either the `HostNetwork` or `NodePortService` endpoint publishing strategy types. The PROXY protocol enables the load balancer to preserve the original client addresses for connections that the Ingress Controller receives. The original client addresses are useful for logging, filtering, and injecting HTTP headers. In the default configuration, the connections that the Ingress Controller receives only contain the source address that is associated with the load balancer.
+A cluster administrator can configure https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[the PROXY protocol] when an Ingress Controller uses either the `HostNetwork` or `NodePortService` endpoint publishing strategy types. The PROXY protocol enables the load balancer to preserve the original client addresses for connections that the Ingress Controller receives. The original client addresses are useful for logging, filtering, and injecting HTTP headers. In the default configuration, the connections that the Ingress Controller receives only contain the source IP address that is associated with the load balancer.
 
-This feature is not supported in cloud deployments. This restriction is because when {product-title} runs in a cloud platform, and an IngressController specifies that a service load balancer should be used, the Ingress Operator configures the load balancer service and enables the PROXY protocol based on the platform requirement for preserving source addresses.
+[IMPORTANT]
+====
+For a passthrough route configuration, client pods in an {product-title} cluster cannot access their original client source IP address from within the cluster. Ensure that you configure Ingress access logging for your Ingress Controller so that you can view what source IP addresses client pods can access from within your cluster.
```

Comment:
A few corrections:

- This issue doesn't only apply to client pods; it applies to all client connections.  
- It isn't the client that would read the source IP address; it's the server pod.  
- Using term "access" could be misunderstood to imply that the pod cannot connect to the IP address; maybe "observe" would be clearer?
- As there is some cost to recording and storing access logs, I believe it would be better to instruct the cluster-admin to enable access logs only if needed.
```suggestion
For a passthrough route configuration, servers in an {product-title} cluster cannot observe the original client source IP address. If you need to know the original client source IP address, ensure that you configure Ingress access logging for your Ingress Controller so that you can view the client source IP addresses.  
```

Reply:
:heavy_check_mark: